packages/comms/src/ecl/logicalFile.ts

```diff
@@ -72,14 +74,15 @@ export class LogicalFile extends StateObject<FileDetailEx, FileDetailEx> impleme
     get IsCompressed(): boolean { return this.get("IsCompressed"); }
     get BrowseData(): boolean { return this.get("BrowseData"); }
     get jsonInfo(): string { return this.get("jsonInfo"); }
-    get binInfo(): string { return this.get("binInfo"); }
+    get binInfo(): base64Binary { return this.get("binInfo"); }
```

Comment:
This was one of those synonymous number types, but I'm not sure if this is right?